### PR TITLE
Don't crash if no root semantics node

### PR DESCRIPTION
**Pull request by abarth**
Fixes #4849

### CODE REVIEW

**Comment by abarth** [comment]
@tvolkert

File: sky/shell/platform/ios/framework/Source/accessibility_bridge.mm [hunk 1]
```diff
@@ -315,8 +315,9 @@ - (BOOL)accessibilityPerformMagicTap {
     }
   }
 
-  if (!view_.accessibilityElements) {
-    view_.accessibilityElements = @[ objects_[kRootNodeId] ];
+  SemanticObject* root = objects_[kRootNodeId];
```
Comment (tvolkert) [inline sky/shell/platform/ios/framework/Source/accessibility_bridge.mm]: Under what conditions do we receive an initial update (i.e. not having set view_.accessibilityElements yet) not containing root?

Comment (abarth) [inline sky/shell/platform/ios/framework/Source/accessibility_bridge.mm]: Suppose someone empties out the render tree and there's nothing on screen.

**Comment by abarth** [comment]
Updated to clear out the list of accessible elements if there is no root element.

File: sky/shell/platform/ios/framework/Source/accessibility_bridge.mm [hunk 2]
```diff
@@ -315,8 +315,13 @@ - (BOOL)accessibilityPerformMagicTap {
...
+  if (root) {
+    if (!view_.accessibilityElements && root) {
```
Comment (tvolkert) [inline sky/shell/platform/ios/framework/Source/accessibility_bridge.mm]: s/ && root//

**Comment by tvolkert** [comment]
lgtm

**Pull request merged**